bioblend/galaxy/toolshed/__init__.py

```diff
@@ -144,3 +144,36 @@ def install_repository_revision(self, tool_shed_url, name, owner,
         url = "%s%s" % (self.gi.url, '/tool_shed_repositories/new/install_repository_revision')
 
         return self._post(url=url, payload=payload)
+
+    def uninstall_repository_revision(self, name, owner, changeset_revision,
+                                      tool_shed_url, remove_from_disk=True):
+        """
+        Uninstalls a specified repository revision from this Galaxy instance.
+
+        :type name: str
+        :param name: The name of the repository
+
+        :type owner: str
+        :param owner: The owner of the repository
+
+        :type changeset_revision: str
+        :param changeset_revision: The revision of the repository to be installed
+
+        :type tool_shed_url: str
+        :param tool_shed_url: URL of the Tool Shed from which the repository was
+                              installed from (e.g., ``https://testtoolshed.g2.bx.psu.edu``)
+
+        :rtype: dict
+        :return: If successful, a dictionary with a message noting the removal
+        """
+
+        payload = {}
+        payload['tool_shed_url'] = tool_shed_url
+        payload['name'] = name
+        payload['owner'] = owner
+        payload['changeset_revision'] = changeset_revision
```

Comment:
I think the changeset_revsion might be optional ? Can you add a test for this ?

Reply:
There wasn't a test for installation that I saw in the current tests. @nsoranzo do we want to add that? I can't do it this week but maybe next

Reply:
If ephemeris is going to test for this, I guess it's fine with no tests here.

Reply:
@mvdbeek I don't think `changeset_revision` is optional, see https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py#L643-L647